src/models/codex-record-metadata.js

```diff
@@ -105,13 +108,18 @@ schema.set('toJSON', {
 
 schema.methods.generateMintTransactionData = function generateMintTransactionData() {
 
+  // @TODO: sort out proper provider ID functionality
+  const providerData = codexRecordService.encodeProviderData([
```

Comment:
I don't think we should call this `providerData` since `data` in the contract could later contain non-provider related stuff (maybe more token metadata? who knows).

Reply:
This would require renaming a bunch of stuff in this review

Reply:
Well my thinking was that since this is data specific to (and defined by) the provider, that `providerData` was fairly apt. You can put any data you want in there and it's still provider data, no? It's not data _about_ the provider, but data _specified_ by the provider... doesn't matter what they choose to put in there.

I hesitate to make this variable name so generic, because that makes it a little less descriptive and harder to do project-wide searches for.

Reply:
Would `mintData` be any better?

Reply:
Ahh, except the same data is passed for `modifyMetadataHashes` call so it's not really mint data in that case... 🤔

Reply:
I'm not against `mintData` but then we would need a different one for the `Modified` event. What about `additionalData`? Still generic, but it makes it a little bit more searchable. I'm just not sold on the `provider` naming convention yet and don't want to propagate it everywhere until we have a better idea of what the decentralization picture looks like.

Reply:
Alright, I've renamed `providerData` to `additionalData` in 6e080a6